### ISSUE TITLE
Application slowdown when posing over extended durations

### ISSUE THREAD

**Issue by Ani-ki** [imchillin/Anamnesis#272]
It would appear that loading .cmp files causes the tool to slow down considerably. We still have yet to find the actual link to prove this theory, despite it being the one thing in common amongst reports. Can only investigate and get additional log files to find a pattern. <s>thank christ this is still in beta</s>

**Comment by Ani-ki** [comment]
Perhaps we can revisit the idea of building a separate tool to batch convert `.cmp` files to `.pose` and likewise `.cma` to `.chara`. Not sure if this could be built in as a separate tab, or if I should just make a standalone tool like my TTMP extractor?

**Comment by Featherbrew** [comment]
Found slowdown in the tool when using a .cma file, attached to a Junkmonger.
Only other thing out of the norm besides posing was delimiting the camera and moving it using the tool, but it felt like there was an element of slowing beforehand. Logs attached.
[2021-04-22-00-20-30.txt](https://github.com/imchillin/Anamnesis/files/6352509/2021-04-22-00-20-30.txt)
[2021-04-22-01-29-26.txt](https://github.com/imchillin/Anamnesis/files/6352510/2021-04-22-01-29-26.txt)

**Comment by Featherbrew** [comment]
Potential lagg source - huge hits of lagg when I partially loaded (Right arm -> Select Children -> Include Selected Bones load) in-built pose files (Miqo'te Fem Idle Standing poses)
[2021-04-23-14-30-04.txt](https://github.com/imchillin/Anamnesis/files/6364767/2021-04-23-14-30-04.txt)

**Comment by Yuuki-Walsh** [comment]
This seems to be happening even without loading poses (see #324) so I'm consolidating the discussion here to any slowdowns that happen while posing.

**Comment by Ani-ki** [comment]
Now looking into the possibility that this is being caused by actor refresh being triggered repeatedly whilst in GPose, which shouldn't be happening.

**Comment by Lynx8891** [comment]
I've noticed this slowdown everytime I use Anamesis. It does not seem to be directly due to loading poses, as it can happen over time. After an extended period, it can slow down to point that it starts jumping around (bones rotate way more than intended), at which point, I'll save all the poses to temp files, close the app and restart. After reloading the poses, the app maintains its speed for a time, but generally seems like it will slow down sooner than the prior start. It almost seemed like maybe excessive ram memory was in use. So I did an experiment where I started it fresh and watched my memory usage. 

Initially with the game and app running (and windows and all the other background stuff of course), just starting a pose I had 6gb of ram allocated, after a time, maybe about 30-40 minutes, I saw my ram usage hit 8gb (maxed on my system). Thinking this was the issue I upgraded to 32gb, and retried it. Started at 6gb like normal, but eventually the app slowed down, and it read 11gb usage, a lot, but not maxed out. Basically I'm seeing Anamesis gradually increase the amount of ram it needs, but does not seem to be the exact issue of the slow down, but I feel like its worth mentioning. Im not programmer, but feel like maybe its a sign of whats going on.
